lib/config/presets/internal/config.ts

```diff
@@ -5,7 +5,7 @@ import type { Preset } from '../types';
 export const presets: Record<string, Preset> = {
   'best-practices': {
     configMigration: true,
-    description: 'Preset with best practices from the Renovate maintainers.',
+    description: 'Preset with best practices from the Renovate maintainers. Recommended for advanced users, who want to follow our best practices.',
```

Comment:
`config:recommended` says:

> Recommended configuration for most users. It does not matter what programming language you use.

Do we want to match that pattern here?

```suggestion
    description: 'Recommended for advanced users, who want to follow our best practices.',
```